samza-core/src/main/java/org/apache/samza/zk/ZkBarrierForVersionUpgrade.java

```diff
@@ -36,44 +36,33 @@
  * This class creates a barrier for version upgrade.
  * Barrier is started by the participant responsible for the upgrade. (start())
  * Each participant will mark its readiness and register for a notification when the barrier is reached. (waitFor())
- * If a timer (started in start()) goes off before the barrier is reached, all the participants will unsubscribe
+ * If a timer (started in start()) goes off before the barrier is reached, all the participants will un-subscribe
```

Comment:
err.. yeah..I think I was blindly following IntelliJ's spelling suggestion. Thanks! I will fix it.